pkg/synopsysctl/cmd_deploy.go

```diff
@@ -115,15 +116,15 @@ var deployCmd = &cobra.Command{
 			return nil
 		}
 
-		// Deploy synopsys-operator
-		log.Debugf("creating Synopsys-Operator components")
+		// Deploy Synopsys Operator
+		log.Debugf("creating Synopsys Operator components")
 		soperatorSpec := soperator.NewSOperator(deployNamespace, synopsysOperatorImage, exposeUI, adminPassword, postgresPassword,
-			userPassword, blackduckPassword, secretType, operatorTimeBombInSeconds, dryRun, logLevel, threadiness, postgresRestartInMins,
+			userPassword, blackDuckPassword, secretType, operatorTimeBombInSeconds, strings.ToUpper(dryRun) == "TRUE", logLevel, threadiness, postgresRestartInMins,
```

Comment:
Also, this check looks pretty ugly in passing of parameters.  This should be moved right above, or change the input type of the function.